### DISCUSSION TITLE
Output format and report

### DISCUSSION THREAD

**Discussion by ChrisRG711** [jtesta/ssh-audit#313] [Q&A] [answered]
Greetings,

I am using SSH-Audit as part of a script that takes an input of server ip addresses and scans each out. It gives great information however i am trying to get the end result of all IPS into a pretty report. HTML or PDF. has anyone else tried this?

**Comment by jtesta** [comment, marked as answer]
Providing pretty reports is outside the scope of this project.
 However, an external tool could be made to take the JSON output and
create any document you'd desire.  I don't know of any pre-made tool
that does this, however.
